configure.ac

```diff
@@ -366,3 +366,9 @@ AC_OUTPUT()
 if [ test "`nc-config --version`" == "netCDF 4.7.4" ]; then
   AC_MSG_RESULT([Warning: netCDF v4.7.4 is loaded. Incompatible fms2_io tests will be skipped and there may be netCDF issues with model runs.])
 fi
+
+# Check if gcc is 11.1 for class(*) select type bug
+if [ test -n "`$FC --version | grep GNU | grep 11.1`" ]; then
```

Comment:
Do we know that 11.0 is safe?  I would think it best to just not support the 11 stream until we know 11.0 and 11.2 have the bug/bugfix.

Reply:
I agree, it's updated now to warn on any 11 versions.

Reply:
I'm not sure if there even was an 11.0 release: http://gcc.gnu.org/releases.html